src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewRenderer.php

```diff
@@ -236,7 +236,7 @@ private function createServerAttributes(PortalInformation $portalInformation, Re
     {
         // get server parameters
         $server = [];
-        $host = $this->defaultHost;
+        $host = null;
```

Comment:
Why is it set at all if it is just set to `null`?

Reply:
the default was only used for websocket

Reply:
the default was only used for websocket